src/System.Private.ServiceModel/src/System/ServiceModel/Description/XmlSerializerOperationBehavior.cs

```diff
@@ -128,6 +128,23 @@ void IOperationBehavior.AddBindingParameters(OperationDescription description, B
 
         void IOperationBehavior.ApplyDispatchBehavior(OperationDescription description, DispatchOperation dispatch)
         {
+            if (description == null)
+                throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull("description");
+
+            if (dispatch == null)
+                throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull("dispatch");
+
+            if (dispatch.Formatter == null)
+            {
+                dispatch.Formatter = (IDispatchMessageFormatter)CreateFormatter();
+                dispatch.DeserializeRequest = _reflector.RequestRequiresSerialization;
+                dispatch.SerializeReply = _reflector.ReplyRequiresSerialization;
+            }
+
+            if (_reflector.Attribute.SupportFaults && !dispatch.IsFaultFormatterSetExplicit)
```

Comment:
This is ported from Desktop code. I've removed the following because wrapper is not used at all:
```c#
else
                {
                    var wrapper = dispatch.FaultFormatter as IDispatchFaultFormatterWrapper;
                    if (wrapper != null)
                    {
                        wrapper.InnerFaultFormatter = (IDispatchFaultFormatter)CreateFaultFormatter(dispatch.FaultContractInfos);
                    }
                }
```